open-ce/build_tree.py

```diff
@@ -252,16 +252,26 @@ def _create_all_recipes(self, variants):
                 if _make_hash(package) in packages_seen:
                     continue
 
+                # If the feedstock value starts with https: or git@, treat it as a url. Otheriwse
+                # combine with git_location and append "-feedstock.git"
+                if package['feedstock'].startswith("https:") or package['feedstock'].startswith("git@"):
```

Comment:
Should this also check for `http:` to make it a little more generic? In case the repo isn't on github.

Reply:
Good idea. I added that to the list.